bin/bundle

```diff
@@ -24,16 +24,14 @@ m = Module.new do
   def cli_arg_version
     return unless invoked_as_script? # don't want to hijack other binstubs
     return unless "update".start_with?(ARGV.first || " ") # must be running `bundle update`
-
     bundler_version = nil
     update_index = nil
     ARGV.each_with_index do |a, i|
       if update_index && update_index.succ == i && a =~ Gem::Version::ANCHORED_VERSION_PATTERN
         bundler_version = a
       end
-      next unless a =~ /\A--bundler(?:[= ](#{Gem::Version::VERSION_PATTERN}))?\z/o
-
-      bundler_version = Regexp.last_match(1)
+      next unless a =~ /\A--bundler(?:[= ](#{Gem::Version::VERSION_PATTERN}))?\z/
```

Comment:
Performance/ConstantRegexp: Extract this regexp into a constant, memoize it, or append an `/o` option to its options.